<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mapmaker</class>
 <widget class="QWidget" name="Mapmaker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1344</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>1031</width>
     <height>771</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="CenterLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>140</y>
     <width>167</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QLabel" name="label_10">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>战斗机数：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="CarrierLineEdit">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="FighterLineEdit">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_11">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>侦察机数：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>潜艇数：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="DestroyerLineEdit">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>运输舰数：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>航母数：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="ScoutLineEdit">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="SubmarineLineEdit">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>驱逐舰数：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="CargoLineEdit">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>10</y>
     <width>168</width>
     <height>108</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>地图大小：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="SizeCombo">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <item>
       <property name="text">
        <string>30*30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>40*40</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>50*50</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>60*60</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>70*70</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>80*80</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>90*90</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>100*100</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>陆地比例：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="LandCombo">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <item>
       <property name="text">
        <string>少</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>中</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>多</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>资源数：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="SourceCombo">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <item>
       <property name="text">
        <string>少</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>中</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>多</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>对称方式：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="CenterCombo">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <item>
       <property name="text">
        <string>左右对称</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>上下对称</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>中心对称</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>120</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>一方参数(各单位数0-10)：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>后退</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HelpButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>350</y>
     <width>191</width>
     <height>66</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1">
     <widget class="QLineEdit" name="PopulationLineEdit">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>60</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="RoundLineEdit">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>200</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>最大回合数(1-1000)：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_12">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>最大人口数(30-200)：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>430</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>天气（-1~1）：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="WeatherLineEdit">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>430</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OpenPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>460</y>
     <width>91</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="ChangeRadioButton">
      <property name="text">
       <string>修改地形</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="FighterButton">
      <property name="text">
       <string>增添战斗机</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="ScoutButton">
      <property name="text">
       <string>增添侦察机</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="CarrierButton">
      <property name="text">
       <string>增添航母</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="DestroyerButton">
      <property name="text">
       <string>增添驱逐舰</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="CargoButton">
      <property name="text">
       <string>增添运输舰</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="SubmarineButton">
      <property name="text">
       <string>增添潜艇</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="OilfieldButton">
      <property name="text">
       <string>增添油田</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="MineButton">
      <property name="text">
       <string>增添矿井</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="FortButton">
      <property name="text">
       <string>增添据点</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="BaseButton">
      <property name="text">
       <string>增添基地</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="DeleteButton">
      <property name="text">
       <string>删除单位</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="SaveButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>保存为</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RandomButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>随机生成</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>540</y>
     <width>55</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QRadioButton" name="Team0Button">
      <property name="text">
       <string>队伍0</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="Team1Button">
      <property name="text">
       <string>队伍1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="Team2Button">
      <property name="text">
       <string>队伍2</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="NewButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ClosePushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>SizeCombo</tabstop>
  <tabstop>LandCombo</tabstop>
  <tabstop>SourceCombo</tabstop>
  <tabstop>CenterCombo</tabstop>
  <tabstop>SubmarineLineEdit</tabstop>
  <tabstop>DestroyerLineEdit</tabstop>
  <tabstop>CarrierLineEdit</tabstop>
  <tabstop>CargoLineEdit</tabstop>
  <tabstop>FighterLineEdit</tabstop>
  <tabstop>ScoutLineEdit</tabstop>
  <tabstop>PopulationLineEdit</tabstop>
  <tabstop>RoundLineEdit</tabstop>
  <tabstop>WeatherLineEdit</tabstop>
  <tabstop>ExitButton</tabstop>
  <tabstop>HelpButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
